src/tokenizer_test.cc

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "src/tokenizer.h"
```

Comment:
I think the old code does it in the way that the style guide requests: Test code for a header should have its include first.